tests/phpunit/tests/html-api/wpHtmlTagProcessor.php

```diff
@@ -72,6 +72,25 @@ public function test_has_self_closing_flag_matches_input_html( $html, $flag_is_s
 		}
 	}
 
+	/**
+	 * @ticket 6.9.0
```

Comment:
Should be the ticket number:
```suggestion
	 * @ticket 63854
```